packages/origin-247-certificate/src/offchain-certificate/offchain-certificate.service.ts

```diff
@@ -9,31 +9,32 @@ import {
 } from './repositories/CertificateEvent/CertificateEvent.repository';
 import { EventBus } from '@nestjs/cqrs';
 
-import {
-    IClaimCommand,
-    IIssueCommand,
-    ITransferCommand,
-    IIssuedCertificate,
-    IIssueCommandParams
-} from '../types';
+import { IClaimCommand, IIssueCommand, ITransferCommand, IIssueCommandParams } from '../types';
 
 import {
     CertificateIssuedEvent,
     CertificateTransferredEvent,
     CertificateClaimedEvent,
     ICertificateEvent
 } from './events/Certificate.events';
+
 import { CertificateCommandEntity } from './repositories/CertificateCommand/CertificateCommand.entity';
+import {
+    CertificateReadModelRepository,
+    CERTIFICATE_READ_MODEL_REPOSITORY
+} from './repositories/CertificateReadModel/CertificateReadModel.repository';
+import { CertificateAggregate } from './certificate.aggregate';
 
-type CertificateCommand = IIssueCommand<unknown> | ITransferCommand | IClaimCommand;
 @Injectable()
 export class OffchainCertificateService<T = null> {
```

Comment:
Just asking - is this interface compatible with CertificateService? I was thinking about this and I think, that they should be compatible. Of course if it's not possible, then not

Reply:
Yes, it's the same